isotovideo

```diff
@@ -246,6 +250,25 @@ $bmwqemu::vars{TEST_GIT_HASH} = checkout_git_refspec($bmwqemu::vars{CASEDIR} =>
 # set a default distribution if the tests don't have one
 $testapi::distri = distribution->new;
 
+my $specfile = path($bmwqemu::vars{PRODUCTDIR} . '/wheels.yaml');
+if (-e $specfile) {
+    my $schema_file = dirname(__FILE__) . '/schema/Wheels-01.yaml';
```

Comment:
Since you're already using `Mojo::File`, it can also export a `curfile` function that could be used here (`curfile->dirname`).

Reply:
I actually changed that to `$installprefix` since the file is somewhere else when it's installed.